docs/design-auth_policies.html

```
@@ -187,7 +496,7 @@ <h1>Authorization Policies</h1>
     <div class="col-md-8">
         
         <p>Link will enforce developer constructed authorization policies to ensure that users have the proper access required before performing any requests. This will help prevent users from having the ability to perform CRUD operations on service data or configurations without having the necessary permissions set by the facility administrators to do so. Authorization policies will be enforced both at the API Gateway and at each individual service.</p>
-<h1 id="request-authorization-sequence">Request Authorization Sequence <a id="request-authorization-sequence" href="#request-authorization-sequence" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Request Authorization Sequence"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h1>
+<h1 id="request-authorization-sequence" class="content-heading">Request Authorization Sequence <a id="request-authorization-sequence" href="#request-authorization-sequence" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Request Authorization Sequence"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h1>
```

HTMLHint (1.9.2)

[error] The id value [ request-authorization-sequence ] must be unique.

(id-unique)